privacyidea/lib/auditmodules/containeraudit.py

```diff
@@ -64,6 +63,7 @@ def log(self, param):
         """
         for module in self.write_modules:
             module.log(param)
+        self.have_data = True
```

Comment:
Assume we have no writeable modules, then it would contain no data - right?
So I think it should be

~~~~python
for module in self.write_modules:
    module.log(param)
    self.have_data = True
~~~~

Reply:
I'll look into it

Reply:
Done in 1997585